Sources/_StringProcessing/Regex/DSLTree.swift

```diff
@@ -164,6 +164,96 @@ extension DSLTree {
       indirect case subtraction(CustomCharacterClass, CustomCharacterClass)
       indirect case symmetricDifference(CustomCharacterClass, CustomCharacterClass)
     }
+    
+    public struct AsciiBitset {
+      let isInverted: Bool
+      let isCaseInsensitive: Bool
```

Comment:
What do you mean?

Reply:
I don't think we can fold `isInverted` into the value, since that needs to match every non-ASCII character in addition to the ASCII characters that aren't included.

Reply:
Oops, I was already folding in the value for isCaseInsensitive, I just forgot to remove the stored property. And yea as Nate said we can't fold isInverted into the value